sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpRetryOptions.java

```diff
@@ -194,8 +194,7 @@ public boolean equals(Object obj) {
 
         final AmqpRetryOptions other = (AmqpRetryOptions) obj;
 
-        return this.getMaxRetries() == other.getMaxRetries()
-            && this.getMode() == other.getMode()
+        return this.getMaxRetries() == other.getMaxRetries() && this.getMode() == other.getMode()
```

Comment:
nit: I liked the previous formatting better as it kept all the conditions in new lines. Now, couple of them are in the same line and the rest are in new lines.

Reply:
I believe this and the conditionals should be doable